.github/workflows/ci.yml

```
@@ -398,9 +398,8 @@ jobs:
           prepare: |
             pkg_add cmake
           run: |
-            cmake -B build
-            cmake --build build -j $(sysctl -n hw.ncpu)
-            ./build/qjs -qd
+            make
+            make stats
```

Just a hunch but make is probably bsd make, not gnu make:
```suggestion
            gmake
            gmake stats
```

---

Yeah it is indeed. TIL `ifeq` is a GNU-ism 😅